### PR TITLE
Set up GitBook

### LINKED ISSUE #140: Set up GitBook

### PR DESCRIPTION
**Pull request by kantoniak**
Sets up GitBook for the repo. I also moved notes about testing to check text formatting etc. See demo [here](https://kantoniak.github.io/gitbook-test-2/). Docs will update each time someone merges changes in `docs/` to the main branch.

Closes #140.

Note that this will overwrite:
1. presentation published [here](https://prinz-nussknacker.github.io/prinz/),
1. presentation files in the `pages` branch.

I suggest switching website branch to `gh-pages` to keep the slides. `gh-pages` is the default pages branch, but we use a different value for some reason. We could also do one of these:
- rename `pages` to something more meaningful like `presentation` (note that there will be at least one more presentation),
- archive slides in a PDF or something similar,
- remove old files all together :smile:.

I would opt for removing presentation from the repo since it's not really about Prinz and won't bring value to visitors.
Thoughts?

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: |
+          npm install gitbook-cli
+          node_modules/gitbook-cli/bin/gitbook.js build
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs/_book
+          commit_message: "Update docs on push"
+          publish_branch: pages  # default: gh-pages
```
Comment (avan1235) [inline .github/workflows/docs.yaml]: this comment  here makes no sense to me

**Review by avan1235 — APPROVED** [review]

**Comment by avan1235** [comment]
The presentation content should be moved to some gitbook page imo and then removed as you described (it should not be a lot of work as it was mostly written in markdown like language). For me you can change the name of published website branch

File: .github/workflows/docs.yaml (hunk 1 above)
Comment (kantoniak) [reply .github/workflows/docs.yaml]: It came copied from the configuration examples. Now I removed this line.

**Comment by kantoniak** [comment]
@avan1235 I did as you said, the presentation is in the `pages` branch.

**Pull request merged**